### PR TITLE
[Docs] Convert "config cheat sheet" to a man page

### PR DESCRIPTION
**Pull request by 6543**
follow up of  #13901

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -709,7 +709,13 @@ generate-manpage:
 	@mkdir -p man/man1/ man/man5
 	@./gitea docs --man > man/man1/gitea.1
 	@gzip -9 man/man1/gitea.1 && echo man/man1/gitea.1.gz created
-	@#TODO A smal script witch format config-cheat-sheet.en-us.md nicely to suit as config man page
+	@#TODO A smal script witch format config-cheat-sheet.en-us.md nicely to suit as man page
+	@tail -n +36 docs/content/doc/advanced/config-cheat-sheet.en-us.md > man/man5/gitea.app.ini.5.md
```
Comment (6543) [inline Makefile:713]: :point_up: dynamical determine what to cut?

**Comment by 6543** [comment]
Idear: "...walk the source code to read the comments and tags of the settings module, and generate `config-cheat-sheet.en-us.md` out of it..."

**Comment by wxiaoguang** [comment]
Does Gitea still need this? The "config cheat sheet" seems too complex to be used as man page?

**Comment by 6543** [comment]
I still would like to have it as man page ...

**Comment by wxiaoguang** [comment]
Which app does so? IMO most tools just put the CLI arguments into man page. I don't see any rich web app put its documents into man page.

**Comment by 6543** [comment]
well I can convert this into a script and move it into contrib instead

it's important for those who want to package gitea

**Pull request closed without merging**